doc/howto/storage_volumes.md

```diff
@@ -29,6 +29,10 @@ To create a custom storage volume with the content type `block`, add the `--type
 
     lxc storage volume create <pool_name> <volume_name> --type=block [configuration_options...]
 
+For creating a custom ISO volume, the following command needs to be used:
```

Comment:
```suggestion
To create a custom storage volume of type `iso`, use the `import` command instead of the `create` command:
```

Reply:
Can you change the first sentence in this section (line 21) to:
```
Use the following command to create a custom storage volume of type `block` or `filesystem` in a storage pool:
```

And then move the new content  (line 32-35) to the end of the section (after the note) - since everything before it is for `filesystem` and `block` only.